### PR TITLE
Add User Offers export

### PR DESCRIPTION
**Pull request by SukiCZ**
Hi, Yesterday MKM [annouced](https://www.cardmarket.com/en/Magic/News/Exporting-User-Offers-Through-The-Cardmarket-API) new endpoint for exporting UserOffers.

https://api.cardmarket.com/ws/documentation/API_2.0:Export_User_Offers

This PR adds functionality to `create_export_user_offers` and `get_export_user_offers`.

Should I add unit-test and docs?

### CODE REVIEW

**Comment by silvanocerza** [comment]
> Should I add unit-test and docs?

That would be great. 🙏

**Comment by SukiCZ** [comment]
@silvanocerza Integration tests and documentation added.

I haven't run the integration tests as [Sandbox server was abandoned](https://api.cardmarket.com/ws/documentation#Sandbox_Server_Abandoned)

The documentation is missing few functions, I'm not sure if mentioning this function in section "Getting started" is appropriate.

I also had to bump few dependencies (python 3.7 reached End Of Life on [Tuesday 27th Jun](https://endoflife.date/python)) to be able to run `tox` and build `docs`.

File: mkdocs.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ theme: readthedocs
 repo_url: https://github.com/evonove/mkm-sdk
 site_description: A simple SDK for dedicated apps for Magic Kard Market.
 site_author: Silvano Cerza
-pages:
+nav:
```
Comment (SukiCZ) [inline mkdocs.yml:6]: Command `mkdocs build` raised an error

```
ERROR    -  Config value 'pages': The configuration option 'pages' was removed from MkDocs. Use 'nav' instead.
Aborted with 1 Configuration Errors!
```

**Comment by SukiCZ** [comment]
@silvanocerza any feedback?

IMHO we should only merge the first commit as tests and docs are missing some basic functionality and adding the latest makes it more obvious/chaotic.

Thanks

**Comment by silvanocerza** [comment]
Thanks so much, this is really a well done job. 🙏 
Sorry I didn't come back to this sooner but I completely forgot.
I'm merging and making a new release right away.

**Pull request merged**